src/core/api/index.js

```diff
@@ -26,14 +32,22 @@ function callApi(endpoint, schema, params) {
     .then((response) => normalize(response, schema));
 }
 
-export function search({ page, query }) {
+export function search({ page, query, state }) {
```

Comment:
Is there something you'd prefer?

I think the state is the right thing to use. It is a little weird that we're passing all of it. Perhaps I could add another API reducer that listens to `SET_JWT` and `SET_LOCALE` (when implemented) and anything else we need and collects them all into an object.

```js
import { search } from 'core/api';

function loadSearch({params: {page, query}, store: {getState}}) {
  return search({page, query, apiConfig: getState().apiConfig});
}